.github/CONTRIBUTING.md

```diff
@@ -330,15 +331,55 @@ what you wanted by clicking the `View deployment` or `Visit Preview` buttons on
 This will take you to a preview of the documentation changes.
 This preview is created by [Vercel](https://vercel.com/docs/getting-started-with-vercel).
 
-## 🏭 Release Process
+## 📕 Releases & Versioning
 
 As of now, LangChain has an ad hoc release process: releases are cut with high frequency by
-a developer and published to [PyPI](https://pypi.org/project/langchain/).
+a developer and published to [PyPI](https://pypi.org/project/langchain/). 
```

Comment:
Shouldn't this be pointing to pypi pages for each package?

Reply:
cut with high frequency by a *maintainer*